spec/adapters/googlecharts/display_spec.rb

```diff
@@ -131,6 +131,110 @@
       expect(js).to match(/containerId: 'id'/)
       expect(js).to match(/view: ''/)
     end
+
+    context 'when formatters are applied in datatables' do
```

Comment:
Please write separate testcases for each formatters, it will be  easy to understand if one of the formatter is breaking . 

I think `formatters` must contains all the information, so isn't we must check the values of it first?

Reply:
Done